resource_lists/haven/resources.yaml

```diff
@@ -0,0 +1,650 @@
+---
+authors:
+  Ben: "https://benjaminvreeland.com"
+
+index_page_display_name: Haven
+
+game_version: "1.1.302"
+
+note: |
+  Based on
+  https://haven.fandom.com/wiki/Items
+  https://steamdb.info/app/983970/history/
+  https://steamcommunity.com/sharedfiles/filedetails/?id=2310841844
+
+recipe_types:
+  Cooking: Cook {IN_ITEMS} into {OUT_ITEM}
+  Cooking (Camping): Cook {IN_ITEMS} into {OUT_ITEM} (only when camping)
+  Synthesizer: Synthesize {IN_ITEMS} into {OUT_ITEM}
+  Crafting: Craft {IN_ITEMS} into {OUT_ITEM}
+  Enhancement: Use {IN_ITEMS} to enhance {OUT_ITEM}
+  Repair: Use {IN_ITEMS} to repair the Nest ({OUT_ITEM})
+
+resources:
+
+  ##############################################################################
+  _a: {H1: Food} ###############################################################
+  ##############################################################################
+
+  _b: {H2: Kitchen} ############################################################
+
+  Roasted appledews:
+    recipes:
+    - output: 1
+      recipe_type: Cooking
+      requirements:
+        Appledew: -10
+    - recipe_type: Raw Resource
+
+  Glazed boba:
+    recipes:
+    - output: 1
+      recipe_type: Cooking
+      requirements:
+        Appledew: -5
+        Boba nut: -5
+    - recipe_type: Raw Resource
+
+  Creamberry flambé:
```

Comment:
I think this should have a `custom_simplename` of `creamberryflambe` so that the associated image name makes more sense, thoughts?

Reply:
I forgot `custom_simplename` was even a thing, I just made it as it is to get it working, but having the simple name makes more sense.